quakemigrate/core/lib.py

```diff
@@ -41,30 +48,30 @@
 
 
 @util.timeit()
-def migrate(onsets, traveltimes, first_idx, last_idx, available, threads):
+def migrate(
+    onsets: np.ndarray[float],
+    traveltimes: np.ndarray[int],
+    first_idx: int,
+    last_idx: int,
+    available: int,
+    threads: int,
+) -> np.ndarray[np.double]:
```

Comment:
Yeah, just something I've gotten in to the habit of including. While not strictly typed, you can add type hints like this. Not strictly necessary, but in an ideal world we would eventually migrate to having this universally. It can be really helpful with clarifying whether a function/variable is 'doing too much', too (i.e., it is difficult to figure out the types, or something can be too many different types).

Reply:
Cool, makes sense. Thanks for explaining!